rpm/unofficial-jolla-language-pack-uk.spec

```diff
@@ -0,0 +1,24 @@
+#make sure change these variables to suit your language
+%define CONFLANG uk
+%define LOCNAME uk_UA
+%define RPM_SUFFIX uk
+%define QM_SUFFIX uk
+
+Name: unofficial-jolla-language-pack-%{CONFLANG}
+Version:	0.1.0
```

Comment:
I know it is not documented anywhere, but we used to use the SFOS version here (4.4.0.68 ATM if I am not mistaken), and bump the Release version if a package changed for a single version.